config/locales/et.yml

```diff
@@ -1300,4 +1299,4 @@ et:
     assignment_status: Õppeülesanne (%{status})
   multi_wiki:
     selector_placeholder: Alusta vikidomeeni sisestamist
-...
+...
```

Comment:
Please exclude the newline changes at the end of the locale files.

Reply:
I think this is as a result of taking out the unused code some lines above or do you mean the "..." should be removed entirely?

Reply:
No, this is likely a result of your text editor automatically remove the invisible newline character at the end of the file.

When you run `git diff`, the only changes should be the lines you removed.